src/containers/WorkspaceOptionsMenu.js

```diff
@@ -0,0 +1,18 @@
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { withTranslation } from 'react-i18next';
+import { getContainerId } from '../state/selectors';
+import { WorkspaceOptionsMenu } from '../components/WorkspaceOptionsMenu';
+
+/** Used for connect */
+const mapStateToProps = state => ({
+  containerId: getContainerId(state),
+});
+
+// containerId: getContainerId(state),/
+const enhance = compose(
+  withTranslation(),
+  connect(mapStateToProps, null),
+);
+
+export default enhance(WorkspaceOptionsMenu);
```

Comment:
One thing to call out here is that the ticket makes explicit where in the menu plugins can add options.  I'm not 100% sure what the state of our plugin system is (`withPlugins`, `PluginHook`/`PluginComponents`, etc), so any guidance on what I should be doing wrt hooking this menu (and presumably the other one) with the plugin system would be greatly appreciated.